src/main/java/thestonedturtle/partypanel/ui/PlayerBanner.java

```diff
@@ -53,47 +54,74 @@
 public class PlayerBanner extends JPanel
 {
 	private static final Dimension STAT_ICON_SIZE = new Dimension(18, 18);
-	private static final Dimension ICON_SIZE = new Dimension(Constants.ITEM_SPRITE_WIDTH, Constants.ITEM_SPRITE_HEIGHT);
+	private static final Dimension ICON_SIZE = new Dimension(Constants.ITEM_SPRITE_WIDTH-6, Constants.ITEM_SPRITE_HEIGHT-4);
```

Comment:
I had noticed this before but never tried fixing it - the icons were being horizontally stretched for some reason and require hard-coding some values to get it sized appropriately. By doing this and decreasing the margin a tiny bit, I can fit the full word "(level-xxx)" even with very long RSNs.